knowledge.md

```diff
@@ -20,6 +24,8 @@
 ### Web Application ###
 4. Describe what SQL Injection is and how you would test for it? *@jstnkndy*
 5. What about Blind SQL Injection and how is it different from other kinds? *@jstnkndy*
+6. How can SQL Injection lead to remote code execution?

```

Comment:
Can you please move this to the open.md file since there are multiple methods of doing it?